sync/sync.go

```diff
@@ -263,6 +263,7 @@ Register:
 		time.Sleep(5 * time.Second)
 		return err
 	}
+	s.SetPreload(true)
```

Comment:
set this to above the 'if' condition.. there is a return from there

Reply:
done